frontend/src/components/pages/AccessControl/index.tsx

```diff
@@ -202,11 +282,24 @@ const AccessControlPage = (): JSX.Element => {
                 />
               </Td>
               <Td>
-                <IconButton
-                  aria-label="Mark as active button"
-                  icon={<FaEllipsisV />}
-                  variant=""
-                />
+                <Popover>
+                  <PopoverTrigger>
+                    <IconButton
+                      aria-label="Mark as active button"
+                      icon={<FaEllipsisV />}
+                      variant=""
+                    />
+                  </PopoverTrigger>
+                  <PopoverContent width="inherit">
+                    <PopoverBody
+                      as={Button}
+                      bg="background.white.100"
+                      onClick={(e) => handleStatusChangePopoverTrigger(user)}
+                    >
+                      Mark as {user.active ? Filter.INACTIVE : Filter.ACTIVE}
```

Comment:
nit: I think the end of the popover should be aligned with the ellipsis icon, and the text should be styled a bit differently?
Current:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/40839109/187587145-6b66f972-88e1-46c4-afb8-40191deb58f5.png">

Design:
<img width="176" alt="image" src="https://user-images.githubusercontent.com/40839109/187587167-e12ef153-262d-4a02-91bc-99defa15f0ee.png">

Reply:
Made it to match the design styling but the font still looks rly big because we needa scale down the font sizes as well

Reply:
Gonna merge anyway and we'll just need to update the textStyles when we get the updated sizes

Reply:
yup yup sounds good